### DISCUSSION TITLE
Newby question: Pages collection seems to be missing in clean install

### DISCUSSION THREAD

**Discussion by dolphiq-allard** [statamic/cms#10046] [Q&A] [answered]
Hey guys,

I am sorry for probably a silly question. Been developing in Craft for years now, and slowly flirting with Statamic because of the Laravel framework. Got it up and running using a Docker setup, however, in the clean install the `Pages` collection seem to be missing.

Since this is missing, there is no way I can create a homepage for my site. Furthermore, if I manually add a collection called 'Pages' and add some entries within, I am missing the 'Live preview' and 'Visit URL' button. 

So I thing there is a system setting somewhere that needs to be adjusted in order to get going.

Any help or a push in the right direction would be awesome.

Cheers,
Allard

**Comment by jackmcdade** [comment, marked as answer]
Sounds like you probably just need to set a route in the pages collection settings. You probably want {parent_uri}/{slug}.

**Reply by dolphiq-allard** [reply]
Thanks for the quick reply. Seems to be working. Will continue exploring Statamic. Thanks again!

**Comment by jackmcdade** [comment]
Did you make it an ordered collection? If so, put the home page in the top of the tree to make it Home.
